zyte_common_items/processors.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, List
+
+from lxml.html import HtmlElement
+from parsel import Selector
+from zyte_parsers import Breadcrumb as zp_Breadcrumb
+from zyte_parsers import extract_breadcrumbs
+
+from .items import Breadcrumb
+
+
+def breadcrumbs_processor(value: Any, page: Any) -> Any:
+    def _from_zp_breadcrumb(value: zp_Breadcrumb) -> Breadcrumb:
+        return Breadcrumb(name=value.name, url=value.url)
+
+    if isinstance(value, (Selector, HtmlElement)):
+        base_url = getattr(page, "url", None)
```

Comment:
WebPage class also has base_url attribute

Reply:
Do you mean we should use `base_url` if it's present and `url` otherwise? Or only `base_url`?

Reply:
I'd actually do an isinstance check